### PR TITLE
KAZOO-5141: move kapi_ modules to kazoo_amqp to fix startup of crossb…

### PR DESCRIPTION
**Pull request by mark2600**
…ar without acdc source

### CODE REVIEW

File: core/kazoo_amqp/src/api/kapi_acdc_queue.erl [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 -export([queue_size/2, shared_queue_name/2]).
 
--include("acdc.hrl").
+-include("../../applications/acdc/src/acdc.hrl").
```
Comment (lazedo) [inline core/kazoo_amqp/src/api/kapi_acdc_queue.erl]: is this include necessary ?

Comment (mark2600) [inline core/kazoo_amqp/src/api/kapi_acdc_queue.erl]: yes, unfortunately

Comment (lazedo) [inline core/kazoo_amqp/src/api/kapi_acdc_queue.erl]: we have to figure out a way to decouple this. we can't have core referring an app

Comment (jamesaimonetti) [inline core/kazoo_amqp/src/api/kapi_acdc_queue.erl]: Why can't kapi_acdc_queue stay in the acdc application?

Comment (lazedo) [inline core/kazoo_amqp/src/api/kapi_acdc_queue.erl]: the problem is that crossbar acdc modules use kapi_acdc but now we have deployments that have only crossbar, thus kapi_acdc is not installed.

this brings back the discussion on queue/agent api abstraction and having apps handling them, acdc or qubicle or other, which is my preferred solution.

**Pull request closed without merging**